net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardClient.xml

```diff
@@ -24,12 +24,12 @@
         <help>Shared secret (PSK) for this peer.</help>
     </field>
     <field>
-        <id>client.tunneladdress</id>
-        <label>Tunnel Address</label>
+        <id>client.allowedips</id>
+        <label>Allowed IPs</label>
         <style>tokenize</style>
         <type>select_multiple</type>
         <allownew>true</allownew>
-        <help>List of addresses to configure on the tunnel adapter. Please use CIDR notation like 10.0.0.1/24.</help>
+        <help>List of addresses allowed to pass trough the tunnel adapter. Please use CIDR notation like 10.0.0.1/24.</help>
```

Comment:
and help text change here

Reply:
You are right, the main problem it's just on the frontview, preserving the contiguity with the official nomenclature just to avoid any confusion in the configuration. I just hope that won't cause some confusion to developers instead.